567 robot search timings

fredvd opened the pull request:
Fixes #567.

fredvd committed:
simplify exclude content types from search test

fredvd committed:
Remove whitespace. Fixes #567

ghost commented:
Can one of the admins verify this patch?

mattss commented:
Test this please jenkins

mattss commented on src/ploneintranet/suite/tests/acceptance/search.robot:
```diff
@@ -80,11 +80,6 @@ I can follow the search result ${SEARCH_RESULT_TITLE}
-    Unselect Checkbox  css=input[type="checkbox"][value="${CONTENT_TYPE}"]
-    Select From List By Value  css=select[name="created"]  today
-    Wait Until Element is Visible  css=dl.search-results[data-search-string*="created=today"]
-
```
Can I ask why this keyword was removed? It now means that the main robot test doesn't read like a user story. Abstracting the actual interactions into a keyword means that when the UI implementation changes, the keywords can be updated and the test cases can stay the same.

gyst commented:
@fredvd I agree with Matt. Can you please factor out the specifics into keywords in lib, then call those keywords?

mattss commented:
They keywords were already in place, so this should just be a case of reverting the 'simplify' commit.

fredvd committed:
Return to verbose abstractions

fredvd commented:
@matts I remove the keyword because the extra filtering on created today items is what's probably causing the random failures and is not part of the test description. That left only one statement but I didn't know it isn't allowed to to explicit selects etc. in the main test descriptions. I've cleaned up the first keyword and added an extra line that checks for 'does not contain'. There's a similar new builtin keyword in selenium 1.7, but we're using 1.6 afaik.

mattss commented:
@fredvd Looks good - thanks!

mattss commented:
Test this please jenkins
